tools/xabuild/xabuild.csproj

```diff
@@ -1,35 +1,24 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+<Project>
+  <!--NOTE: manually importing Sdk.props and Sdk.targets so we can replace the Pack target-->
+  <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.props" />
```

Comment:
Same comment for `<Project Sdk="Microsoft.NET.Sdk" />`

Reply:
Commit revereted. This project will not be migrated to new SDK style.